egress/egress-only-tracking.yaml

```diff
@@ -0,0 +1,74 @@
+components:
+  schemas:
+    EgressOnlyTacking:
+      description: |-
+        The list of structures containing Egress offset and it's length to be placed in the packet.
+      type: object
+      required: [port_name]
+      properties:
+        port_name:
+          description: Name of the Rx port where egress only tracking enabled.
+          type: string
+          x-constraint:
+          - '/components/schemas/Port/properties/name'
+          x-field-uid: 1
+        metric_tags:
+          description: |-
+            One or more metric tags can be used to enable tracking portion of or all bits
+            in a corresponding header field for metrics per each applicable value.
+            These would appear as tagged metrics in corresponding egress_only_tracking metrics.
+          type: array
+          items: 
+            $ref: '#/components/schemas/EgressOnlyTacking.MetricTag'
+          x-field-uid: 2
+        filter:
+          description: |-
+            Offset in bits where the tracking value will be placed in the packet.
```

Comment:
"Specified a rule which will be used to first filter received packets on this port before applying the egress tracking metric_tags on them. If the packet does not match the rule, it will not be considered for egress tracking."  -> Again might not more accuracy but definitely current description is not appropriate.